OpenShift/helm/nexus-repository-manager/README.md

```diff
@@ -0,0 +1,262 @@
+| `deployment.annotations`                    | Annotations to enhance deployment configuration  | `{}`                       |
+| `deployment.initContainers`                 | Init containers to run before main containers  | `nil`                        |
+| `deployment.postStart.command`              | Command to run after starting the nexus container  | `nil`                    |
+| `deployment.additionalContainers`           | Add additional Container         | `nil`                                      |
+| `deployment.additionalVolumes`              | Add additional Volumes           | `nil`                                      |
+| `deployment.additionalVolumeMounts`         | Add additional Volume mounts     | `nil`                                      |
+| `secret.enabled`                            | Enable secret                    | `false`                                    |
+| `secret.mountPath`                          | Path to mount the secret         | `/etc/secret-volume`                       |
+| `secret.readOnly`                           | Secret readonly state            | `true`                                     |
+| `secret.data`                               | Secret data                      | `nil`                                      |
+| `service.enabled`                           | Enable additional service        | `nil`                                      |
+| `service.name`                              | Service name                     | `nil`                                      |
+| `service.portName`                          | Service port name                | `nil`                                      |
+| `service.labels`                            | Service labels                   | `nil`                                      |
+| `service.annotations`                       | Service annotations              | `nil`                                      |
+| `service.loadBalancerSourceRanges`          | Service LoadBalancer source IP whitelist | `nil`                              |
+| `service.targetPort`                        | Service port                     | `nil`                                      |
+| `service.port`                              | Port for exposing service        | `nil`                                      |
+| `route.enabled`         | Set to true to create route for additional service | `false` |
+| `route.name`            | Name of route                                      | `docker` |
+| `route.portName`        | Target port name of service                        | `docker` |
+| `route.labels`          | Labels to be added to route                        | `{}` |
+| `route.annotations`     | Annotations to be added to route                   | `{}` |
+| `route.path`            | Host name of Route e.g jenkins.example.com         | nil |
+
+
+```bash
+$ helm install --name my-release --set persistence.enabled=false stable/sonatype-nexus
```

Comment:
for reference, I'd value seeing this example with *2*  configs set, just so I don't need to lookup how to apply multiples.